vectorlm/utils/model_utils.py

```diff
@@ -194,6 +194,7 @@ def _module_init_fn(module: nn.Module) -> Callable:
     ret_dict["auto_wrap_policy"] = auto_wrap_policy
     ret_dict["sharding_strategy"] = sharding_strategy
     ret_dict["device_id"] = torch.cuda.current_device()
+    ret_dict["forward_prefetch"] = True
```

Comment:
Is this related to the FSDP profiling?

Reply:
Yes, but not entirely. The CUDA syncs that we saw in Jacob's table turned out to be time spent of global CUDA syncs. Regardless, this option should allow for full communication overlap in the forward pass.